src/searchForm/datePicker/datePickerTopBar/index.tsx

```diff
@@ -0,0 +1,25 @@
+import styled from 'styled-components'
+import { color, space } from '_utils/branding'
+
+import DatePickerTopBar from './DatePickerTopBar'
+
+const StyledDatePickerTopBar = styled(DatePickerTopBar)`
+  & {
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+    margin-bottom: ${space.l};
+  }
+
+  & .kirk-datePickerTopBar-text {
+    color: ${color.primaryText};
+    padding-left: ${space.m};
+  }
+
+  & .kirk-datePickerTopBar-button {
+    width: auto;
+    height: auto;
```

Comment:
Will sync with Jonas about this, because we introduced a new type of button which is basically inline and unstyled